README.md

```diff
@@ -37,7 +37,11 @@ document.addEventListener("tokenReceived", e => {
 
 ## Publish to npm
 
-<!-- Dave.js: I'm not sure if we just do the regular publishing or if there's a job for it liek we do for UI. -->
+1. Add a git tag with the version number you want to publish. 
+   - For example, version 1 would be `git tag v1.0.0`.
+1. Publish the version to npm by running `npm run publish`. 
```

Comment:
Did you have to run `npm run publish --public` when you did it the second time? 🤔

Reply:
oh right, hmm. nope, it didn't tell me to do this. maybe it just needed to be done the first time since it hadnt been published before